bolt-design-system/bolt
# Increase Incache Config Max Age + Adjust Cache Behavior When Deploying

## Jira
http://vjira2:8080/browse/BDS-889?filter=-1

## Summary
Updates the `incache` plugin used internally for grabbing and caching the different versions / releases of Bolt to expire after 30 days (vs every 5 minutes). Addresses annoying file changes while doing local development + updates Travis behavior to bypass this cache when doing a tagged Bolt release.

## How to test
- Confirm the .incache file doesn't update when pulling in these updates locally and running the build

> Note: testing the deploy-related behavior with this will need to get tested separately unless @EvanLovely had any ideas on this
